src/gui/editorwidgets/qgsvaluerelationwidgetwrapper.h

```diff
@@ -20,7 +20,8 @@
 #include "qgsvaluerelationfieldformatter.h"
 
 #include <QComboBox>
-#include <QListWidget>
+#include <QTableWidget>
+#include <QHeaderView>
```

Comment:
I think these includes should go to the cpp file (and a forward declaration in here).

Reply:
ok